src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx

```diff
@@ -122,494 +122,153 @@ Here's how you can access the distributed tracing UI, depending on the type of s
 
 We have a variety of tools to help you find traces and spans so you can resolve issues. The opening distributed tracing page is populated with a default list of traces, and you can quickly refine this list using these tools:
 
-* [Query bar filters](#query-bar)
-* [Left-pane filters](#left-pane-filters)
-* [Trace groups](#trace-groups)
-* [Scatter plot chart](#scatter-plot)
-* [Interactive charts](#trace-charts)
+<Tabs stacked>
```

Comment:
I wonder if the tabs component is the best way to present this information. It seems like these are currently grouped as different ways you can interact with the UI to find "useful traces". That's a bit vague, and I think that vagueness probably leads to a very dense h2 with a lot of content.

I wonder how this doc would read if you unspooled this section and broke up the content into 2 different h2s, e.g.:

h2. Organize your span data 

Intro content
Collapser 1: Query your span data (idk if that's the "action" for this section 
Collapser 2: Group similar traces

h2. Filter your span data 
Collapser 3: Filter your span data (select left-pane filters + interact with trace charts)
Collapser 4: View anomalous spans with scatter plots

Reply:
This isn't like a prescriptive suggestion, just an idea for different ways to organize this info given that the side tab component is dense! I don't think it looks bad as is though!